src/store/cart-store.js

```diff
@@ -20,10 +25,9 @@ store.registerHandler('CHECKOUT_ITEM_FOUND', data => {
     store.emitChange();
 });
 
-
 store.registerHandler('CHECKOUT_SUCCESS', () => {
     store.waitFor(StudentStore);
-    items = [];
+    store.clearItems();
```

Comment:
Or in this case, we do not need to use an action and should simply clear items in the previous fashion.